store/store.go

```diff
@@ -80,31 +82,31 @@ type (
 	}
 )
 
-func cleanDateString(dateString string) (string, error) {
+func cleanDateString(dateString string) (time.Time, error) {
+	date := time.Time{}
+
 	if dateString == "" {
-		return "", nil
+		return date, nil
 	}
+
 	date, err := time.Parse(time.RFC3339Nano, dateString)
 	if err != nil {
-		return "", err
+		return date, err
 	}
 
-	// The Golang implementation of time.RFC3339Nano does not use a fixed number of digits after the
-	// decimal point and therefore is not reliably sortable. And so we use our own custom format for
-	// database range queries that will properly sort any data with time stored as an ISO string.
-	// See https://github.com/golang/go/issues/19635
-	return date.Format(RFC3339NanoSortable), nil
+	return date, nil
 }
 
 // GetParams parses a URL to set parameters
 func GetParams(q url.Values, schema *SchemaVersion) (*Params, error) {
 
-	startStr, err := cleanDateString(q.Get("startDate"))
+	// NOTE: *Str is redundant once dates are all converted to native dates
```

Comment:
Should this comment be removed?

Reply:
possibly, i was expecting to note all double date handling so it could be cleaned up easier at a later date once no longer needed, but the number of locations made it mostly white noise to keep doing, ill remove them.